### ISSUE TITLE
Ability to repost with comment

### ISSUE THREAD

**Issue by cleverdevil** [cleverdevil/together#16]
Presently, Together offers users the ability to reply, like, or repost an item. It would be wonderful if there was a way to repost with comment, with the ability to enter more longform comments. One of my personal primary ways to share content is to "bookmark" it in my website with some commentary. Some examples:

* https://cleverdevil.io/2017/apples-new-ad-for-the-ipad-pro-is-pure-genius
* https://cleverdevil.io/2017/daring-fireball-cultural-insularity-and-apple-tv

Especially nice would be the ability to embed block quotes, as I do in the second example above.

**Comment by aaronpk** [comment]
"repost with comment" isn't a post type itself. Anything consuming posts that sees a `repost-of` property will not look at that post for any content. This might be more appropriate as a bookmark post, which can definitely contain content, or even just a regular post that happens to link to the original post, the way Quote Tweets work.

**Comment by cleverdevil** [comment]
Yes, that's precisely what I mean, @aaronpk. In fact, both examples above are created as `u-bookmark-of` posts. I'd also be OK with the way Quote Tweets work.

**Comment by cleverdevil** [comment]
I started on this in the [wip-bookmark-of branch](https://github.com/cleverdevil/together/compare/wip-bookmark-of). It actually seems to work just find, creating content on the micropub endpoint, but for some reason I get an error. Not sure why.

I could use an extra set of smarter eyes at some point :)

**Comment by grantcodes** [comment]
I've not copied this over to the new repo yet because I'm not too sure what to do with it...

I am kind of against the idea of calling this a bookmark in the UI because I think it might cause confusion, I would have thought replying to the post would be enough? Then the rest is up to your site if it shows more context with the reply or not.

A different type of bookmark might be a possibility though
